.github/workflows/pull-request-title-validation.yml

```diff
@@ -25,12 +25,11 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Validate PR title
+        env:
+          PR_TITLE: ${{ github.event.pull_request.title }}
         run: |
           # Regex for Conventional Commits specification
-          PR_TITLE_REGEX="^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test)(\([^\)]*\))?:\s?(EXPOSED-[0-9]+\s?)?.+$"
-
-          # Get the PR title
-          PR_TITLE="${{ github.event.pull_request.title }}"
+          PR_TITLE_REGEX="^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test)(!)?(\([^\)]*\))?:\s?(EXPOSED-[0-9]+\s?)?.+$"
```

Comment:
Modified regular expression to allow for optional "!" (to denote breaking changes) after the commit type